js1/uvod-do-js/cvlekce/prevod-meny.md

```diff
@@ -6,8 +6,8 @@ solutionAccess: lock
 ---
 
 1. Založte si novou prázdnou stránku s JavaScriptovým programem.
-1. Dejme tomu, že si jako programátoři vyděláváte 20 euro na hodinu. Uložte tuto hodnotu do proměnné `wageInEur`.
-1. Spočítejte, kolik je vaše hodinová mzda v českých korunách, jestliže kurz eura je `26.58` Kč. Výsledek zaokrouhlete na celé koruny a uložte do proměnné :var[wageInCzk].
+1. Dejme tomu, že si jako programátoři vyděláváte **20 euro na hodinu**. Uložte tuto hodnotu do proměnné `wageInEur`.
```

Comment:
Mě by se líbilo, kdyby ta cvičení **nebyla skrz na skrz prošpikovaná** tím ztučněním. Přijde mi to jako **CAPS LOCK**, že to na mě řve v každé větě a když je to opravdu v **každé větě**, tak si jednak připadám **jako debil**, že na mě někdo něco pořád křičí, ale taky **to začnu ignorovat**. Proč se třeba tady na mě křičí, že **vydělávám 20 euro!!!!!!**? Myslím, že účastnice jsou **dostatečně schopné** tu informaci zaznamenat i bez **toho zvýraznění**.